src/cmds/scripts/pbs_habitat.in

```diff
@@ -797,6 +797,8 @@ if [ "${PBS_START_SERVER:-0}" != 0 ] ; then
 	# Source the file that sets PGSQL_LIBSTR
 	. "$PBS_EXEC"/libexec/pbs_pgsql_env.sh
 
+	PBS_licensing_loc_file=PBS_licensing_loc
```

Comment:
It looks like pbs_habitat expects to find this file in PBS_EXEC/etc which is fine if it is packaged in the RPM, but I don't believe it is. A better place for this file would be PBS_HOME/server_priv. The /opt/pbs directory won't get fully removed if the RPM is uninstalled because the directory structure contains a file that was not part of the original RPM.